libs/output-mapping/src/DeferredTasks/LoadTableQueue.php

```diff
@@ -33,7 +34,15 @@ public function __construct(Client $client, array $loadTableTasks)
     public function start()
     {
         foreach ($this->loadTableTasks as $loadTableTask) {
-            $loadTableTask->startImport($this->client);
+            try {
+                $loadTableTask->startImport($this->client);
+            } catch (ClientException $e) {
+                throw new InvalidOutputException(
```

Comment:
Jako tady bych asi navrhoval spíš
if ($e->getCode() < 500) throw new InvalidOutputException
else throw $e;

Reply:
Mel jsem za to ze `ClientException` se vyhazuje jen u 4XX a jinak probubla `ServerException` z Guzzle. Koukam ale do zdrojaku a vidim ze to tak neni. Tak to doplnim!.